files/install-deps-worker.yaml

```diff
@@ -2,6 +2,15 @@
 - name: Install dependencies for packit-service worker.
   hosts: all
   tasks:
+    - name: Download packit spec file
+      get_url:
+        url: https://raw.githubusercontent.com/packit-service/packit/master/packit.spec
+        # this should be improved - zuul is using var peoject_dir - can be used also in non-zuul
+        # runs - if not set -> set?, will be there any zuul runs conflicts?
+        dest: ./packit.spec
+    - name: Install packit RPM build dependencies
+      command: dnf install $(rpmspec -q --requires packit.spec | grep -v 'packit')
```

Comment:
Thanks, but there's actually needed more than this:
See https://github.com/packit-service/research/pull/52#discussion_r448998138 and `rpm -q --requires python3-packit`.

Maybe the ['local packit rpm build' idea](https://github.com/packit-service/research/pull/52#discussion_r449002033) would be the easiest to implement after all. Have you tried that?

Reply:
i tried local build -  it required to install build deps using `dnf builddep file.spec`
so i cloned packit-service/packit and installed builddeps  using `dnf builddep file.spec`,  build rpm using `rpmbuild -ba --build-in-place packit.spec --define "_rpmdir ."`  then i was comparing what `dnf install $(rpmspec -q --requires packit.spec| grep -v packit)` and `rpm -q --requires python3-packit` want to install using locally build rpms:
`dnf install $(rpmspec -q --requires packit.spec| grep -v packit)` wanted to install `python3-packit dependencies` + fedpkg, python, rpmdevtools.

Not 100% sure if all of this additional packages are required (fedpkg, python, rpmdevtools), but in any case this is most straightforward approach:

`dnf builddep file.spec`
`dnf install $(rpmspec -q --requires packit.spec| grep -v packit)`

this 2 commands installed all require dependencies. 

WDYT?

Reply:
I see the following in `rpm -q --requires python3-packit`:
```
python3.8dist(munch)
python3.8dist(pygithub)
python3.8dist(requests)
```
they are not part of `dnf builddep packit.spec` or `rpmspec -q --requires packit.spec`.

I'm wondering whether we should add them as `BuildRequires`?

Reply:
i just checked that 
`python3.8dist(munch)`
`python3.8dist(pygithub)`
`python3.8dist(requests)`
are installed with `dnf builddep file.spec`

Reply:
They are pulled in by other modules:
`pygithub` -> `python3-ogr`, `munch`&`requests` -> `python3-copr`
```
repoquery -q --requires python3-copr
repoquery -q --requires python3-ogr
```

I see [munch](https://github.com/packit-service/packit/blob/master/packit/copr_helper.py#L30) and [requests](https://github.com/packit-service/packit/blob/master/packit/distgit.py#L31) used in code, so I'd add those two.
I don't see `pygithub` used in code so I'd try to remove it from [setup.cfg](https://github.com/packit-service/packit/blob/master/setup.cfg#L34).

Reply:
created PR for this https://github.com/packit-service/packit/pull/899